.env.webcomponent.example

```diff
@@ -6,3 +6,4 @@ PUBLIC_URL=http://localhost:3012
 REACT_APP_IDENTITY_URL='http://localhost:3011'
 REACT_APP_API_ENDPOINT='http://localhost:3009'
 ASSETS_URL='http://localhost:3011'
+PUBLIC_URL = "http://localhost:3011"
```

Comment:
🔧 `PUBLIC_URL` is already defined so needs updating on line 5

Reply:
Ah, yes, had missed this! Should be sorted now 👍